src/ProgressMeter.jl

```diff
@@ -70,11 +70,28 @@ function next!(p::Progress)
     end
 end
 
+function next!(p::Progress)
```

Comment:
Now calling just `next!` will never update the progress bar, meaning that it will break existing code.